mne/io/eyelink/calibration.py

```diff
@@ -0,0 +1,203 @@
+"""Eyetracking Calibration(s) class constructor."""
+
+# Authors: Scott Huberty <seh33@uw.edu>
+# License: BSD-3-Clause
+
+from collections import OrderedDict
+from ...utils import fill_doc
+
+
+@fill_doc
+class Calibrations(list):
```

Comment:
Rather than `Calibrations` I think it would be nicer to have `Calibration` that subclasses `dict` (basically just a `dict` with a nice `__repr__`) and have `read_calibrations` return `list of Calibration`

Reply:
Oh I didn't read far enough that `Calibrations` does have `Calibration` as its entries... I'm not sure there's enough value added to this `Calibrations` class over just `Calibration`. Let's stick with bare `list of Calibration` as the return type to keep things simpler

Reply:
Ok!